package/MDAnalysis/coordinates/DCD.py

```diff
@@ -90,440 +86,162 @@
-        else:
-            self.step = step
-            self.delta = delta
-        self.dcdfile = open(self.filename, 'wb')
-        self.remarks = remarks
-        self._write_dcd_header(self.n_atoms, self.start, self.step, self.delta, self.remarks)
-
-    def _dcd_header(self):
-        """Returns contents of the DCD header C structure::
-             typedef struct {
-               fio_fd fd;                 // FILE *
-               fio_size_t header_size;    // size_t == sizeof(int)
-               int natoms;
-               int nsets;
-               int setsread;
-               int istart;
-               int nsavc;
-               double delta;
-               int nfixed;
-               int *freeind;
-               float *fixedcoords;
-               int reverse;
-               int charmm;
-               int first;
-               int with_unitcell;
-             } dcdhandle;
-
-        .. deprecated:: 0.7.5
-           This function only exists for debugging purposes and might
-           be removed without notice. Do not rely on it.
-
-        """
-        # was broken (no idea why [orbeckst]), see Issue 27
-        # 'PiiiiiidiPPiiii' should be the unpack string according to the struct.
-        #    struct.unpack("LLiiiiidiPPiiii",self._dcd_C_str)
-        # seems to do the job on Mac OS X 10.6.4 ... but I have no idea why,
-        # given that the C code seems to define them as normal integers
-        desc = [
-            'file_desc', 'header_size', 'natoms', 'nsets', 'setsread', 'istart',
-            'nsavc', 'delta', 'nfixed', 'freeind_ptr', 'fixedcoords_ptr',
-            'reverse', 'charmm', 'first', 'with_unitcell']
-        return dict(zip(desc, struct.unpack("LLiiiiidiPPiiii", self._dcd_C_str)))
-
-    def write_next_timestep(self, ts=None):
-        """Write a new timestep to the DCD file.
-
-        Parameters
+    def __init__(self, filename, convert_units=True, dt=None, **kwargs):
+        """Parameters
         ----------
-        ts : `Timestep` (optional)
-            `Timestep` object containing coordinates to be written to DCD file;
-            by default it uses the current `Timestep` associated with the
-            Writer.
-
-        Raises
-        ------
-        :exc:`ValueError`
-           if wrong number of atoms supplied
-        :exc:`~MDAnalysis.NoDataError`
-           if no coordinates to be written.
-
-
-        .. versionchanged:: 0.7.5
-           Raises :exc:`ValueError` instead of generic :exc:`Exception`
-           if wrong number of atoms supplied and :exc:`~MDAnalysis.NoDataError`
-           if no coordinates to be written.
+        filename : str
+            trajectory filename
+        convert_units : bool (optional)
+            convert units to MDAnalysis units
+        dt : float (optional)
+            overwrite time delta stored in DCD
+        **kwargs : dict
+            General reader arguments.
```

Comment:
Besides the remove of the correl function there aren't any major changes in the function. It should behave as the old implementation